examples/mcu-board-support/stm32h735g.rs

```diff
@@ -38,24 +39,24 @@ pub fn init() {
         .expect("backend already initialized");
 }
 
-#[derive(Default)]
+#[link_section = ".frame_buffer"]
+static mut FB1: [TargetPixel; DISPLAY_WIDTH * DISPLAY_HEIGHT] =
+    [software_renderer::Rgb565Pixel(0); DISPLAY_WIDTH * DISPLAY_HEIGHT];
+#[link_section = ".frame_buffer"]
+static mut FB2: [TargetPixel; DISPLAY_WIDTH * DISPLAY_HEIGHT] =
+    [software_renderer::Rgb565Pixel(0); DISPLAY_WIDTH * DISPLAY_HEIGHT];
+
 struct StmBackend {
-    window:
-        core::cell::RefCell<Option<Rc<slint::platform::software_renderer::MinimalSoftwareWindow>>>,
-    timer: once_cell::unsync::OnceCell<hal::timer::Timer<pac::TIM2>>,
+    window: RefCell<Option<Rc<slint::platform::software_renderer::MinimalSoftwareWindow>>>,
+    scb: RefCell<cortex_m::peripheral::SCB>,
+    delay: RefCell<stm32h7xx_hal::delay::Delay>,
+    timer: hal::timer::Timer<pac::TIM2>,
+    layer: RefCell<stm32h7xx_hal::ltdc::LtdcLayer1>,
+    touch_i2c: RefCell<stm32h7xx_hal::i2c::I2c<stm32h7xx_hal::device::I2C4>>,
```

Comment:
Lots of RefCell.   Would it make sense to have a single `RefCell<StmBackendInner>` or `RefCell<StmBackendDevices>`  or something like that?
Not for performance reason because that doesn't matter much, but just for style purposes as they all get borrowed together.